### ISSUE TITLE
Sort Cards list in advanced search based on cards arrangement defined in card manager

### ISSUE THREAD

**Issue by adamlodge** [archesproject/arches#6987]
Currently the list of cards is sorted alphabetically within advanced search.  Would be better to sort based on how the admin defines in card manager.

![Screen Shot 2021-01-06 at 6.04.28 PM.png](https://images.zenhubusercontent.com/56f150f76caf56c9d38e16f8/37ae3545-c8ec-43cf-a575-bc69e469f8c5)

**Comment by dwuthrich** [comment]
I disagree; for users who don't routinely edit data (and therefore don't know the admin's card order), an alphabetical listing makes more sense.

**Comment by azaroth42** [comment]
This is the same issue as #6332. Propose to close one or the other. 

As far as my thoughts - I would prefer card order, not because of the editor, but because of the order in the report. Normally (I would hope) the most commonly filled out cards will be at the top of the model, to avoid having a bunch of null values before the actual data. So the likelihood that the data is important, and that searches will work, is also in that order. The searching user is (I would also hope) familiar with the report structure.

Alpha is much better than the previous random order, so I don't think this is a high priority, but I'd like to keep it open.

**Comment by chiatt** [comment]
Logic used in #7867 might apply here as well
